src/ServicePulse.Host/app/js/views/endpoint_details/endpoint_details.controller.js

```diff
@@ -32,6 +33,10 @@
             updateUI();
         };
 
+        $scope.formatTime = function (value) {
```

Comment:
I'm not sure if there's a better way to call js functions from a view. Ideally these would be global functions on the `formatter` object, but I'm not sure how to bind the view to something outside of $scope